src/main/java/at/hannibal2/skyhanni/utils/repopatterns/RepoPatternManager.kt

```diff
@@ -156,4 +156,10 @@ object RepoPatternManager {
         }
         return RepoPatternImpl(fallback, key).also { usedKeys[key] = it }
     }
+
+    fun getUnusedPatterns(prefix: String): List<Pattern> {
```

Comment:
Can you add a javadoc here indicating that this is a dangerous method (since it circumvents the exclusivity map)?